### ISSUE TITLE
[russian] com-x.life

### ISSUE THREAD

**Issue by Mormaethor1989** [inorichi/tachiyomi-extensions#472]
(CATALOGUE) SOURCE REQUESTS:
  One source per issue, create more issues if you have more websites
  Check for duplicates, use search (in issues, not the global one)
  Put the name of the site as Title, replace this text with clickable link to the site
BUGS:
  Provide tachiyomi version as well as version of source the report is about

**Comment by Pavkazzz** [comment]
Yes, but there is some problems with content of site, are you a developer of this site?
